<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReportDialog</class>
 <widget class="QDialog" name="ReportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Generate Report</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QDateEdit" name="startDateEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDateEdit" name="endDateEdit">
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
     <property name="displayFormat">
      <string>dd/MM/yyyy</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="generateButton">
     <property name="text">
      <string>Generate Report</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
